### ISSUE TITLE
Can log in with empty user/password field

### ISSUE THREAD

**Issue by smithy545** [CIDARLAB/cello#5]
If you just press login with nothing in the input fields it logs you in as an empty user. You probably have an empty username/password in your database or something. Regardless you should probably do client side checks for that type of stuff.

**Comment by Electra79** [comment]
I also noticed this. It doesn't seem to allow you to use the app if you use an empty login though (after my full 3 minutes of testing)

**Comment by rcjoaquin** [comment]
Yes, you can do login using an emtpy login though. You have a little error list to correct first.

1. If you don't do login, you cannot view About, or any other page.
2. You cannot run any example in the webpage, logged as null/null
3. You haven't got any register page.
4. When you try to install the github project, some libraries are needed, for example "org.cellocad.BU.dom.DWire;"

Please correct them as soon as possible. It seems to be a very interesting project, but if doesn't work properly, nobody cannot see results.

**Comment by rkrishnasanka** [comment]
@rcjoaquin on your point 3. Please look at the screenshot.

![image](https://cloud.githubusercontent.com/assets/855696/14323486/c83b5156-fbee-11e5-94e3-8b0c8ade42e5.png)

**Comment by rcjoaquin** [comment]
Owch...

Sorry @rkrishnasanka, I was blind. ;)
